src/main/java/com/cloudbees/hudson/plugins/folder/computed/ComputedFolder.java

```diff
@@ -234,6 +235,15 @@ protected void submit(StaplerRequest req, StaplerResponse rsp) throws IOExceptio
         }
     }
 
+    @Override
+    protected @Nonnull String getSuccessfulDestination() {
+        FolderComputation<I> computation = getComputation();
+        if (computation != null) {
```

Comment:
It is a little trickier than with `Job`/`Run` where there is a permanent identifier for each run, included in the URL, and they are saved for later perusal. `FolderComputations` are simply overwritten currently (though there is an option to save backup files on disk).